Firestore/Example/Podfile

```diff
@@ -14,7 +14,7 @@ target 'Firestore_Example_iOS' do
   # The next line is the forcing function for the Firebase pod. The Firebase
   # version's subspecs should depend on the component versions in their
   # corresponding podspec's.
-  pod 'Firebase/CoreOnly', '5.20.1'
+  pod 'Firebase/CoreOnly', '5.20.990'
```

Comment:
Nit: `90` instead of `990`? the podspec currently has `90`.

Reply:
Firebase is 990. FirebaseCore is 90.

Reply:
Ahh yep, sorry about that.